app/helpers/bgeigie_imports_helper.rb

```diff
@@ -33,4 +33,14 @@ def operation_button(bgeigie_import, action, text = t(format('.%s', action)))
       f.submit text, class: 'btn btn-primary'
     end
   end
+
+  def unmoderated_id_list(bgeigie_imports)
+    bgeigie_imports.each_with_object([]) do |import, import_url_list|
+      import_url_list << bgeigie_import_path(import) unless import.approved?
```

Comment:
Just occurred to me here that this might catch rejected as well. Will push to staging & see, but we might want to say if `import.submitted?`